lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -269,6 +272,11 @@ && requiresAuthoritativeSourceCheck(newVcs, sessionVcs)) {
         return JOURNEY_NEXT;
     }
 
+    private void setFailedIdentityCheckOnIpvSessionItem(IpvSessionItem ipvSessionItem) {
+        ipvSessionItem.setFailureCode(ReverificationFailureCode.IDENTITY_CHECK_FAILED);
+        ipvSessionItem.setFailureDescription("Identity check failed.");
+    }
```

Comment:
Do you think we should skip this if the scope isn't reverification? I guess it doesn't hurt too much, but could be an extra round trip to DynamoDB.